utils/api/base.js

```diff
@@ -18,15 +18,10 @@ export const fetcher = (url, token) => {
 export const posting = async (url, data, token) => {
   try {
     const response = await api.post(url, data, { headers: defaultHeaders(token) })
-    let quotedWareID = response.data.quoted_ware_refs?.[0].id
-    let requestID = response.data.id
-    if (requestID) {
-      return {
-        success: true,
-        error: false,
-        requestID,
-        quotedWareID,
-      }
+
+    return {
+      data: response.data,
+      error: false,
```

Comment:
simplify `posting` so it is easier to use in other use cases in the future. also, it shouldn't care about the data so much. the function calling it should.

Reply:
this also aligns our return values with the return values from `useSWR`. 😎